static/app/components/dashboards/widgetQueryFields.tsx

```diff
@@ -56,7 +56,20 @@ function WidgetQueryFields({
   function handleAdd(event: React.MouseEvent) {
     event.preventDefault();
 
-    const newFields = [...fields, ''];
+    const newFields = [
+      ...fields,
+      {kind: FieldValueKind.FIELD, field: ''} as QueryFieldValue,
+    ];
+    onChange(newFields);
+  }
+
+  function handleAddEquation(event: React.MouseEvent) {
+    event.preventDefault();
+
+    const newFields = [
+      ...fields,
+      {kind: FieldValueKind.EQUATION, field: ''} as QueryFieldValue,
```

Comment:
Thoughts on putting some default here? Maybe `count() * 2` or something simple like that, so we don't cause a
`Unable to parse your equation, make sure it is well formed arithmetic`
when adding equations